src/fake_bpy_module/transformer/data_type_refiner.py

```diff
@@ -226,43 +275,47 @@ def _get_refined_data_type_fast(
             return [make_data_type_node("typing.Callable")]
 
         if m := REGEX_MATCH_DATA_TYPE_MATHUTILS_VALUES.match(dtype_str):
-            if variable_kind in ('FUNC_ARG', 'CONST', 'CLS_ATTR'):
+            if variable_kind in ("FUNC_ARG", "CONST", "CLS_ATTR"):
```

Comment:
We use single quote for "FUNC_ARG" and so on which are enum.

Reply:
Same as above